lib/logstash/filters/parser.rb

```diff
@@ -37,7 +35,7 @@ def initialize(tokens_list, gramdict, threshold)
   #
   # Returns:
   # A boolean indicating whether the token is dynamic (true) or static (false).
-  def is_dynamic(tokens, dynamic_indices, index)
+  def dynamic_token?(tokens, dynamic_indices, index)
```

Comment:
more conventional way of naming boolean functions in ruby (also gets rid of rubocop warnings :P)